src/main/java/cloud/fogbow/ras/core/plugins/interoperability/googlecloud/compute/v1/GoogleCloudComputePlugin.java

```diff
@@ -0,0 +1,275 @@
+package cloud.fogbow.ras.core.plugins.interoperability.googlecloud.compute.v1;
+
+import cloud.fogbow.common.exceptions.FatalErrorException;
+import cloud.fogbow.common.exceptions.FogbowException;
+import cloud.fogbow.common.exceptions.InternalServerErrorException;
+import cloud.fogbow.common.models.GoogleCloudUser;
+import cloud.fogbow.common.util.PropertiesUtil;
+import cloud.fogbow.ras.core.plugins.interoperability.googlecloud.sdk.v1.compute.models.CreateComputeRequest;
+import cloud.fogbow.ras.core.plugins.interoperability.googlecloud.sdk.v1.compute.models.CreateComputeResponse;
+import static cloud.fogbow.ras.core.plugins.interoperability.googlecloud.sdk.v1.compute.models.CreateComputeRequest.*;
```

Comment:
Better use import, instead of import static, since this will force you to explicitly refer to the static class name in the code, making it clearer that you are invoking a method from a static class. This leaves the code more readable and maintainable.